app/src/models/diff/diff-selection.ts

```diff
@@ -51,15 +51,11 @@ function typeMatchesSelection(
  * whose selection state has diverged from the default selection state.
  */
 export class DiffSelection {
-  private readonly defaultSelectionType:
-    | DiffSelectionType.All
-    | DiffSelectionType.None
-
   /* Any line numbers where the selection differs from the default state. */
-  private readonly divergingLines: Set<number> | null
+  //private readonly divergingLines: Set<number> | null
 
   /* Optional set of line numbers which can be selected. */
-  private readonly selectableLines: Set<number> | null
+  //private readonly selectableLines: Set<number> | null
```

Comment:
How come these are commented out instead of removed?

Reply:
Oh, thanks! Great catch!

It seems that I had forgotten to remove these and move those comments down after testing that the constructor itself was working. I have updated to fix this!

Reply:
thank you both!